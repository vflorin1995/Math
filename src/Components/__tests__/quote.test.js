import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';
import Quote from '../quote';

describe('Jest Snapshot testing suite', () => {
  it('Matches DOM Snapshot', () => {
    const quotes = renderer.create(<Router><Quote /></Router>).toJSON();
    expect(quotes).toMatchSnapshot();
  });
});
