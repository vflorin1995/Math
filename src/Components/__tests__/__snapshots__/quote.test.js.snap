// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest Snapshot testing suite Matches DOM Snapshot 1`] = `
<div>
  <p>
    Pure mathematics is, in its way, the poetry of logical ideas.- Albert Einstein
  </p>
</div>
`;
